pkg/cmd/create/group.go

```diff
@@ -110,7 +111,7 @@ func (o *groupOptions) Run(cmd *cobra.Command, args []string) error {
 		resourceObj := &resource.ResourceObject{
 			Kind:       resource.ResourceTypePrefix + "Group",
 			APIVersion: "2.0",
-			Data:       &directory.Group{},
+			Data:       &openapi.GroupResponseV2{},
```

Comment:
Let's leave the wrapper objects but use composition like so -

```
type Group struct {
       openapi.GroupResponseV2
}
```

This is to allow for flexibility at the module level, but at the same time we leverage the openapi generated code.